traitsui/qt4/editor_factory.py

```diff
@@ -138,9 +138,6 @@ def init(self, parent):
         self.control = QtGui.QLabel(self.str_value)
 
         if self.item.resizable is True or self.item.height != -1.0:
-            self.control.setSizePolicy(
-                QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Expanding
-            )
```

Comment:
I think this code did not make sense.
This class subclasses from `Editor` which defines the following method:
https://github.com/enthought/traitsui/blob/d7315b47e3b1818db28048c047c42a07bbb82b2a/traitsui/qt4/editor.py#L312-L358
This should appropriately handle if resizable or springy is set.  It seems redundant and error prone to reproduce the logic here.
This resolves the original issue.  I am wary this may be a problematic change for UIs with a layout that depends on this (seemingly incorrect) behavior.


I believe the `set_size_policy` code in `Editor` was written after this code here, and now this code is no longer needed.

Reply:
The `set_size_policy` method defined for the simple Enum editor is still needed